### DISCUSSION TITLE
Documentation for developers

### DISCUSSION THREAD

**Discussion by bekogeko** [Pulth-Team/Pulth#19] [Ideas]
Should we add documentation for the whole project? This can help us with the complexity of the system. Also, we can divide the project into smaller pieces. So we can easily remember and rework them. This documentation can be either a simple Notion or a GitHub page.

### Pros:
- easier reworks on the old codebase 
- a quick way to recognize the design decisions
- easily adaptable way for new developers
- easier to collaborate for both backend and frontend teams

### Cons:
- hard to maintain in a long term
- should be updated on every commit*
- takes extra time along with the tasks itself

**Comment by dhincal** [comment]
We can create a documentation but not with complexity. New comers can adapt themselves with a brief documentation not with the complex one. For us I don't think it's necessary to lose time with preparing a giant documentation that we don't really need.
